package.json

```diff
@@ -21,41 +21,42 @@
   "dependencies": {
     "@alrra/travis-scripts": "3.0.1",
     "@google-cloud/storage": "1.6.0",
+    "@octokit/rest": "15.9.5",
     "ajv": "5",
     "babel-core": "6.26.3",
     "babel-eslint": "8.2.3",
     "babel-loader": "7.1.4",
     "babel-plugin-react-intl": "2.3.1",
     "babel-plugin-transform-class-properties": "6.24.1",
     "babel-plugin-transform-flow-strip-types": "6.22.0",
-    "babel-preset-env": "1.6.1",
+    "babel-preset-env": "1.7.0",
     "babel-preset-flow": "6.23.0",
     "babel-preset-react": "6.24.1",
     "babel-preset-stage-2": "6.24.1",
     "chalk": "2.4.1",
     "chromeless": "1.4.0",
     "convict": "4.0.1",
-    "coveralls": "2.13.1",
-    "css-loader": "0.28.11",
+    "coveralls": "3.0.2",
```

Comment:
Do we use / need coveralls in this repo anymore?

Reply:
no